### ISSUE TITLE
[templates] suggestion, split windows vs2019 and vs2022 as separate templates

### ISSUE THREAD

**Issue by dimitre** [openframeworks/openFrameworks#8157]
I'm no regular Windows user, but I think it would be great if we had separate templates and core for vs2019 and vs2022
or another way of choosing between tools v142 / v143

**Comment by ofTheo** [comment]
At this point my feeling is we should only officially support one version of Visual Studio and include in our setup guide how to change the toolset ( which is usually just hitting okay on the dialog box that pops up ). 

Maybe there is another reason for having multiple templates? 
Just nervous about having to support additional things when so little is currently working.

**Comment by roymacdonald** [comment]
I dont think it is a good idea. It just adds to the TO-DO list and I think that  is unnecessary. 
I agree with theo, just to support the newer version and update the setup guide.

**Comment by danoli3** [comment]
I designed the Apothecary stuff to allow for this, and new builds have VS2019 libraries deployed to bleeding as of today. 
- Just x64 

**Todo:**
- Restore VS2019 template with code changes (new cpp files added etc)
- ProjectGenerator add template option

**Comment by roymacdonald** [comment]
Honestly, I don't see the point for doing this. What benefit you see from doing such? The issue I see is that you have one more template to care about, which also reflects on PG, etc.

**Comment by danoli3** [comment]
Yeah probably not needed as vs2022 is going to be prime for a while since no vs2024 is planned. Yet still a lot of devs live in 2019 land and since I already set this up I'll just fix up the PG and clone the latest project configs from 2024 for the 2019 it should still work the same
